sdk/storage/azure-storage-blob/swagger/src/main/java/BlobStorageCustomization.java

```diff
@@ -23,6 +23,11 @@ public void customize(LibraryCustomization customization, Logger logger) {
         PackageCustomization models = customization.getPackage("com.azure.storage.blob.models");
         models.getClass("PageList").addAnnotation("@JsonDeserialize(using = PageListDeserializer.class)");
 
+        models.getClass("BlobCopySourceTags").rename("BlobCopySourceTagsMode");
```

Comment:
Could this be done with a Swagger transform?

Reply:
Is that preferable to a customization?